### PR TITLE
Buddhabhasita Dasabhadra Karmamarga Sutra

### PR DESCRIPTION
**Pull request by JJM317**

### CODE REVIEW

File: _content/canon/buddhabhasita-dasabhadra-karmamarga-sutra_upasaka-wong-moulam.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - "https://drive.google.com/file/d/1fB6KVwWegE3IjRD6bbDWNQ7Zfkq6jXyD/view?usp=drive_link"
+course: karma
+tags:
+  - bodhisatta
+  - merit
+publisher: "Buddha Educational Foundation"
+pages: 8
+---
+
+An English translation of the Buddhabhasita Dasabhadra Karmamarga Sutra (Sutra Spoken by Buddha on the Way of Ten Meritorious Deeds). In this sutra, the Buddha enumerates the ten wholesome actions, or meritorious dharmas, which results benefit all beings.
```
Comment (buddhist-uni) [inline _content/canon/buddhabhasita-dasabhadra-karmamarga-sutra_upasaka-wong-moulam.md:15]: Just the last sentence. And clean it up a bit:

The Buddha enumerates the ten meritorious deeds and their lavish results to a great Naga king.

Comment (buddhist-uni) [reply _content/canon/buddhabhasita-dasabhadra-karmamarga-sutra_upasaka-wong-moulam.md:15]: I might also mention how this sutra builds on EBTs such as AN 10.171.

File: _content/canon/buddhabhasita-dasabhadra-karmamarga-sutra_upasaka-wong-moulam.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+translator: "Upasaka Wong Mou-Lam"
+external_url: "https://ftp.budaedu.org/ebooks/pdf/CE011.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1fB6KVwWegE3IjRD6bbDWNQ7Zfkq6jXyD/view?usp=drive_link"
+course: karma
+tags:
+  - bodhisatta
+  - merit
+publisher: "Buddha Educational Foundation"
+pages: 8
```
[the remark this answers is not in the document]
Comment (JJM317) [reply _content/canon/buddhabhasita-dasabhadra-karmamarga-sutra_upasaka-wong-moulam.md:12]: Pages 1-32 with about 4 per print A$ so about 8. But 10 works too :)

File: _content/canon/t0600.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+translator: "Upasaka Wong Mou-Lam"
+external_url: "https://ftp.budaedu.org/ebooks/pdf/CE011.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1fB6KVwWegE3IjRD6bbDWNQ7Zfkq6jXyD/view?usp=drive_link"
+course: karma
+tags:
+  - bodhisatta
+  - indian
+publisher: "Buddha Educational Foundation"
+pages: 10
```
Comment (buddhist-uni) [inline _content/canon/t0600.md:12]: Should have caught this in review (my bad!) but all works need a year, even if it's a guess.  Here I'll just go ahead and add 1940 as an educated guess (feel free to let me know if you have the exact year!)

**Pull request merged**